### DISCUSSION TITLE
Flask 2.0.3 release

### DISCUSSION THREAD

**Discussion by bluefish6** [pallets/flask#4441] [Q&A] [answered]
Hi,

First of all, thanks for your work on Flask!
I'm sorry to open a new discussion similar to one already answered before (https://github.com/pallets/flask/discussions/4244) :wink:  

I'm wondering if there is any timeline regarding release of version 2.0.3? It seems that last issue addressed in this release was [closed on 23rd December](https://github.com/pallets/flask/milestone/17) and you're already doing great progress on 2.1.0.

I'm asking as I'm affected by https://github.com/pallets/flask/issues/4295 (fixed in 2.0.3) and can confirm that current `master` fixes things for me.

Best

**Comment by tachyondecay** [comment, marked as answer]
The answer is to your question remains the same as in the discussion you referenced.
